python/lsst/jointcal/jointcal.py

```diff
@@ -799,6 +802,8 @@ def _load_data(self, inputSourceTableVisit, inputVisitSummary, associations,
                 visitCatalog = dataRef.get(parameters={'columns': columns})
 
                 selected = self.sourceSelector.run(visitCatalog)
+                if len(selected) == 0:
+                    raise RuntimeError("No data to process: the source selector removed all sources")
```

Comment:
Wait, should this actually raise if one of the visits has no sources?

Reply:
Hhhmmm...good point.  Do you think I should just log a message (INFO/WARNING?) and `continue`?